includes/ini.php

```diff
@@ -144,12 +144,22 @@
     $content_config .= "\$sys_info['php_compress_methods'] = [" . $_compress_method . "];\n";
 
     //server_headers
-    stream_context_set_default(array(
-        'http' => array(
-            'method' => "GET",
-            'header' => "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8\r\nAccept-Encoding: gzip, deflate, br\r\nUser-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:54.0) Gecko/20100101 Firefox/54.0\r\n"
-        )
-    ));
+    if((isset($_SERVER['HTTPS']) and (strtolower($_SERVER['HTTPS']) == 'on' || $_SERVER['HTTPS'] == '1')) || $_SERVER['SERVER_PORT'] == 443) {
```

Comment:
@anhtunguyen hiện tại em đọc chuẩn PSR-12 không thấy nhắc đến quy định dùng toán tử logic `&&`, `||` nhưng đa số trong code mình dùng keyword `and`, `or` thay cho nó nên anh xem chỗ này nên đổi hay cứ dùng thoải mái.

Reply:
Tùy mọi người, qui đinh thế nào anh sẽ làm thế

Reply:
Như vậy thống nhất dùng `and`, `or` luôn nhé anh